povm_toolbox/library/randomized_projective_measurements.py

```diff
@@ -381,7 +400,14 @@ def _get_outcome_label(
             A tuple of indices indicating the POVM outcomes on each qubit. For each qubit,
             the index goes from :math:``0`` to :math:``2 * self.n_PVM - 1``.
         """
-        return tuple(pvm_idx[i] * 2 + int(bit) for i, bit in enumerate(bitstring_outcome[::-1]))
+        # Take into account the possible twirling of the measurements and un-do
+        # its effect. For single-qubit projective measurements, it means to
+        # perform a bit-flip of the classical outcome for each twirled projective
+        # measurement.
```

Comment:
I wonder if this could just be placed in the docstring of this method. But since this is a private method anyways, it does not really matter that much.